### PR TITLE
Cleanup analysis legacy package

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
The package is put into the wrong place, which should be part of the analysis.

### CODE REVIEW

**Review by linsun — COMMENTED** [review]
LGTM assuming tests pass, defer to others to look at it too

**Review by howardjohn — APPROVED** [review]

**Review by linsun — APPROVED** [review]

**Comment by hanxiaop** [comment]
@istio/wg-user-experience-maintainers PTAL, thanks

**Review by GregHanson — APPROVED** [review]
lgtm

**Pull request merged**
